app/views/reservations/new.html.haml

```diff
@@ -21,8 +21,8 @@
 
 .wysiwyg= @instrument.description
 
-- unless @instrument.online?
-  %p.alert.alert-danger= t(".offline_notice")
+- if @instrument.offline? || @instrument.offline_via_schedule_share?
```

Comment:
It's true checking `#offline?` and `#offline_via_schedule_share?` is redundant. I left it for documentation value. Without `#offline?` it gave me the impression that this only has to do with instruments that have shared schedules. A better name for `#offline_via_shared_schedule?` might fix that.